pages/reference/api/overview.md

```diff
@@ -103,6 +102,15 @@ curl -X GET \
 -H "Authorization: Bearer <AUTH_TOKEN>"
 ```
 
+Similarly we can extend our earlier API call that retrieves all applications to also include their device type slug by using a `$expand`:
```

Comment:
Since it's clear from the above (v5 based) example that we intended to show how to retrieve an application w/ its 
device type slug, but we never updated that for v6, I've now added that as example with $expand.